CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
-# Change Log
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
+and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
```

Comment:
comment out this line

Reply:
Done. Although I think I'd prefer if we didn't bother doing this, I'm not sure there's much benefit, and I've noticed we sometimes forget (or at least I do) to *uncomment* the header later. keep-a-changelog's own changelog doesn't do the header comment toggle thing: https://github.com/olivierlacan/keep-a-changelog/blob/master/CHANGELOG.md